commcare_connect/opportunity/models.py

```diff
@@ -406,6 +406,7 @@ class CompletedWork(models.Model):
     entity_id = models.CharField(max_length=255, null=True, blank=True)
     entity_name = models.CharField(max_length=255, null=True, blank=True)
     reason = models.CharField(max_length=300, null=True, blank=True)
+    status_modification_date = models.DateTimeField(auto_now=True)
```

Comment:
This will update every time the model is saved, rather than just when the status is updated. Also, I think it needs to be nullable since older records won't have this data

Reply:
Added null=True